beam/beam/scan/config.py

```diff
@@ -34,10 +34,8 @@ def get_scan_doctypes():
 			scannable_doctypes.add(key)
 			[frm_doctypes.add(value) for value in values.keys()]
 
-	_scan_last = frappe.get_all(
-		"BEAM Settings",
-		["show_scan_output"],
-	)
+	# TODO: should this be filtered against a specific company?
```

Comment:
Associating an arbitrary user to company isn't easy, it requires an Employee record. I think adding a comment on the field is better that it will be visible in all companies.